docs/upgrade.html

```diff
@@ -65,6 +65,12 @@
         producer's <code>batch.size</code> configuration.</li>
 </ul>
 
+<h5><a id="upgrade_1100_new_protocols" href="#upgrade_1100_new_protocols">New Protocol Versions</a></h5>
+<ul>
+    <li> <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-107%3A+Add+purgeDataBefore()+API+in+AdminClient">KIP-107</a>: FetchRequest v5 introduces a partition-level <code>log_start_offset</code> field. </li>
```

Comment:
I think it may be better for owners of KIP-98 to add this in the upgrade note (e.g. in KAFKA-4816) because they know much more about the change in KIP-98 than I do.. Anyway, I can go over the KIP-98 doc and add things that I think necessary.